Tests/Unit/LoRaWan/LoRaLibraryTest.cs

```diff
@@ -3,16 +3,16 @@
 
 #pragma warning disable CA1303 // Do not pass literals as localized parameters
 
-namespace LoRaWan.Tests.Unit.LoRaWanTests
+namespace LoRaWan.Tests.Unit
```

Comment:
A general observation about the tests under `LoRaWan`: why remove the entire `LoRaWanTests` here instead of just the `Tests` suffix? Would it not help discoverability and be a less invasive change if we were having them under a namespace `LoRaWan.Tests.Unit.LoRaWan`?

Reply:
I thought `LoRaWan.Tests.Unit.LoRaWan` doesn't really make sense because it has 2 times `LoRaWan` in there (complies with the requirement to not have repetition there). It is arguable though what's the best approach here, I could also see yours.